cmd/argo/commands/template/create.go

```diff
@@ -42,7 +43,7 @@ func CreateWorkflowTemplates(filePaths []string, cliOpts *cliCreateOpts) {
 	if cliOpts == nil {
 		cliOpts = &cliCreateOpts{}
 	}
-	ctx, apiClient := client.NewAPIClient()
+	ctx, apiClient := client.NewAPIClient(context.Background())
```

Comment:
it would be fine, recommended, to add `ctx context.Context` as the first argument of `CreateWorkflowTemplates` (and any similar functions).
if not, these should probably be `context.TODO()`

Reply:
Done, I think I changed this in all places, so the Cobra context gets provided.